src/components/character-count/threshold/index.njk

```diff
@@ -5,12 +5,22 @@ layout: layout-example.njk
 
 {% from "govuk/components/character-count/macro.njk" import govukCharacterCount %}
 
+{% set textareaValue -%}
+Type more letters into this field after this message to see the threshold feature.
+
+GOV.UK includes services, information and guidance that only government can provide. It’s designed so that users do not need to understand the complex structures of government in order to complete their task. Its content and design is user-centred so that interacting with government is accessible to everybody.
+
+GOV.UK is the official source of government information. It is impartial so that people trust the content they find.
+
+Content and services are continually improved to meet user needs. You can visit the GOV.UK roadmap for the most up to date plan for future improvements.
```

Comment:
I'd recommend for screen reader users that we put the most important info at the top, and then add a note that all the text after a certain point is filler text.

Something like this maybe?

```suggestion
This example of a textarea has a character limit of 1000 characters. The character count is hidden, but will appear when there are more than 850 characters entered into this textarea. Type some more text into this textarea to see the character count appear.

Beyond this sentence, all text in this text area is filler-text, for demonstration purposes.

GOV.UK includes services, information and guidance that only government can provide. It is designed so that users do not need to understand the complex structures of government in order to complete their task. Its content and design is user-centred so that interacting with government is accessible to everybody.

GOV.UK is the official source of government information. It is impartial so that people trust the content they find.
```

Reply:
Oh, in that case we could perhaps just repeat the paragraph, will suggest shortly.

Reply:
Yep that works too! Just something to let screen reader users know that "everything beyond this point is trash" 😛 
And then we can add trash!

Reply:
844 characters:

```suggestion
This example of a textarea has a character limit of 1000 characters. The character count is hidden, but will appear when there are more than 850 characters entered into this textarea. Type some more text into this textarea to see the character count appear. This paragraph is repeated below.
	
This example of a textarea has a character limit of 1000 characters. The character count is hidden, but will appear when there are more than 850 characters entered into this textarea. Type some more text into this textarea to see the character count appear. This paragraph is repeated below.

This example of a textarea has a character limit of 1000 characters. The character count is hidden, but will appear when there are more than 850 characters entered into this textarea. Type some more text into this textarea to see the character count appear.
```